dockstore-integration-testing/src/test/java/io/dockstore/webservice/MetadataResourceIT.java

```diff
@@ -96,4 +96,11 @@ void testCheckHealth() {
 
         Assert.assertThrows(ApiException.class, () -> metadataApi.checkHealth(List.of("foobar")));
     }
+
+    @Test
+    void testEntryTypeMetadataList() {
+        ApiClient anonymousApiClient = getAnonymousOpenAPIWebClient();
+        MetadataApi metadataApi = new MetadataApi(anonymousApiClient);
+        assertEquals(io.dockstore.webservice.core.EntryTypeMetadata.values().size(), metadataApi.getEntryTypeMetadataList().size());
```

Comment:
`EntryTypeMetadata` can be imported, should not be ambiguous, only worth it if there are other changes